common-content/en/module/how-our-curriculum-works/morning-orientation-block/index.md

```diff
@@ -3,19 +3,21 @@ title="Morning orientation"
 emoji='🎡'
 time=10
 [objectives]
-    1="Locate a morning orientation block"
-    2="Check learning objectives for morning orientation block"
+    1="Locate the morning orientation activity in a day plan"
+    2="Check learning objectives for morning orientation activity"
 [build]
   render = 'never'
   list = 'local'
   publishResources = false
 
 +++
 
-We use the morning orientation block to gather the community together. We nominate a time-keeper and a facilitator (if they weren't already nominated during the week). The objectives of the morning orientation block are clear:
+Early in most day plans, we use the morning orientation activity to gather the community together. We nominate a time-keeper and a facilitator (if they weren't already nominated during the week).
```

Comment:
```suggestion
Early in most day plans, we use the morning orientation to gather the community together. We nominate a time-keeper and a facilitator (if they weren't already nominated during the week).
```

Reply:
I don't think you need to say activity this much. It especially becomes a bit confusing given the existence of the Activity Handbook, which doesn't contain this kind of activity. 😉 🫠